queries.sql

```diff
@@ -110,4 +110,43 @@ FROM
     animals
 WHERE
     date_of_birth BETWEEN 'Jan 01,1990'
-    AND 'Dec 31,2000';
\ No newline at end of file
+    AND 'Dec 31,2000';
+
+
+     SELECT a.name 
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+SELECT a.name 
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+SELECT o.full_name, a.name
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id
+GROUP BY o.full_name, a.name;
+
+SELECT s.name, COUNT(a.id) AS animal_count
+FROM animals a
+JOIN species s ON a.species_id = s.id
+GROUP BY s.name;
+
+SELECT a.name 
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+JOIN species s ON a.species_id = s.id
+WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+
+SELECT a.name 
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts=0;
+
+SELECT o.full_name, COUNT(a.id) AS animal_count
+FROM owners o
+JOIN animals a ON o.id = a.owner_id
+GROUP BY o.full_name
+ORDER BY COUNT(a.id) DESC
+LIMIT 1;
```

Comment:
- [ ] Kindly remove the `limit 1` from your queries otherwise you will see only one owner who owns the animals, if you remove it and see the results you will find a table with owners and number of owned pets/animals.